server/src/main/java/org/apache/druid/server/security/TLSUtils.java

```diff
@@ -183,10 +183,12 @@ public static SSLContext createSSLContext(
       KeyStore trustStore = KeyStore.getInstance(trustStoreType == null
                                                ? KeyStore.getDefaultType()
                                                : trustStoreType);
-      trustStore.load(
-          new FileInputStream(trustStorePath),
-          trustStorePasswordProvider == null ? null : trustStorePasswordProvider.getPassword().toCharArray()
-      );
+      try (final FileInputStream trustStoreFileStream = new FileInputStream(trustStorePath)) {
```

Comment:
While at this PR, please replace usages of `FileInputStream`/`FileOutputStream` in the places that you have touched with `Files.newInputStream`/`Files.newOutputStream` when possible. See this comment: https://github.com/apache/incubator-druid/pull/6677#discussion_r237087342.

Reply:
Done.